js/src/components/contentAnalysis/ReadabilityAnalysisLegacy.js

```diff
@@ -0,0 +1,58 @@
+/* global wpseoPostScraperL10n wpseoTermScraperL10n */
+
+import React from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+
+import Results from "./Results";
+
+let localizedData = {};
+if( window.wpseoPostScraperL10n ) {
```

Comment:
Hmm, I think we should find a solution to get rid of this localized global variables outside of components. We've done this as wel for the keyword analysis, so I think we should fix it at a later point.